Find a functionally equivalent code:android.provider.Settings.Secure.putStringForUser:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -1,4 +1,8 @@
 {
+    if (LOCATION_MODE.equals(name)) {
+        // TODO: once b/10491283 fixed, remove this hack
+        return setLocationModeForUser(resolver, Integer.parseInt(value), userHandle);
+    }
     if (MOVED_TO_GLOBAL.contains(name)) {
         Log.w(TAG, "Setting " + name + " has moved from android.provider.Settings.System" + " to android.provider.Settings.Global");
         return Global.putStringForUser(resolver, name, value, userHandle);

